test/core.js

```diff
@@ -95,6 +96,15 @@ describe('Core methods', function() {
             done();
           });
     });
+
+    it('should use syntax from config', function(done) {
+      comb.use(require('./helpers/plugin_syntax_map'))
+        .configure({ syntax: { ".tcss": "scss" }})
```

Comment:
Config should contain configuration for plugin.
For example:
```js
.configure({
  syntax: {
    '.tcss': 'scss'
  },
  'syntax-map': true
})
```
Otherwise plugin's `lint` method will not run.